### PR TITLE
Add streaming_api_base_url to /api/v1/instance

### PR DESCRIPTION
**Pull request by ikuradon**
Added streaming_api_base_url to /api/v1/instance for resolving streaming url from some applications.

### CODE REVIEW

**Comment by ykzts** [comment]
related with #2068

File: app/views/api/v1/instances/show.rabl [hunk 1]
```diff
@@ -5,3 +5,4 @@ node(:title)       { Setting.site_title }
 node(:description) { Setting.site_description }
 node(:email)       { Setting.site_contact_email }
 node(:version)     { Mastodon::Version.to_s }
```
Comment (ykzts) [inline app/views/api/v1/instances/show.rabl]:
```ruby
# ...
node(:email)                  { Setting.site_contact_email }
node(:version)                { Mastodon::Version.to_s }
node(:streaming_api_base_url) { Rails.configuration.x.streaming_api_base_url }
```

adjust the space.

File: app/views/api/v1/instances/show.rabl [hunk 2]
```diff
@@ -1,7 +1,8 @@
...
-node(:title)       { Setting.site_title }
-node(:description) { Setting.site_description }
-node(:email)       { Setting.site_contact_email }
-node(:version)     { Mastodon::Version.to_s }
+node(:uri)                    { site_hostname }
+node(:title)                  { Setting.site_title }
+node(:description)            { Setting.site_description }
+node(:email)                  { Setting.site_contact_email }
+node(:version)                { Mastodon::Version.to_s }
+node(:streaming_api_base_url) { Rails.configuration.x.streaming_api_base_url }
```
Comment (Gargron) [inline app/views/api/v1/instances/show.rabl]: Is this naming convention/structure okay with everybody? Or should we perhaps put this into a "links" or "urls" object?

**Comment by clworld** [comment]
I don't think there is much need to add more URLs in future. (Only few URLs are required for make APIs working)
But wrapping URLs with some block is better design I think.

**Comment by ikuradon** [comment]
Should we use like this?
```
node(:version) { Mastodon::Version.to_s }
node :urls do
    { :streaming_api_base => Rails.configuration.x.streaming_api_base_url }
end
```

**Comment by Gargron** [comment]
@ikuradon Yes, that is good. I think "base" is not necessary

**Review by Gargron — APPROVED** [review]

**Pull request merged**